projects/demo/src/app/pages/search-field-demo/search-field-demo.component.ts

```diff
@@ -14,6 +14,7 @@ const BASE_REQUEST_PATH: string = 'search-field-demo/examples';
 export class SearchFieldDemoComponent {
   public readonly formControl: FormControl = new FormControl('');
   public readonly placeholderControl: FormControl = new FormControl('Placeholder');
+  public readonly maxLengthControl: FormControl<number | null> = new FormControl(null);
```

Comment:
```suggestion
  public readonly maxLengthControl: FormControl<number | null> = new FormContro<number | null>l(null);
```

Еще будет круто если поправишь типы выше, раз уж тут)

Reply:
Поправил)